src/cli/cli.ts

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs');
+const path = require('path');
```

Comment:
I think you can make your imports more consistant with:

```suggestion
import * as fs from 'fs';
import * as path from 'path';
```